concurrency/2.0/apidocs/allclasses-frame.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!-- NewPage -->
+<html lang="en">
+<head>
+<!-- Generated by javadoc (1.8.0_252) on Sat May 30 19:51:36 BST 2020 -->
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<title>All Classes (jakarta.enterprise.concurrent-api 2.0.0-RC2 API)</title>
```

Comment:
Although I see this in your title element here, I don't see a reference to the version anywhere in the rendered javadoc.  It would be good to know what version of the Concurrency javadoc is being referenced when reading the javadoc.  Maybe I missed it, but I can't find it using the rendered url...

Reply:
OK not sure how to do that but will try

Reply:
@smillidge, I experimented a bit with Concurrency and I can't get the version to get produced either.  When the javadoc is generated for Concurrency and I display the Index.html, I don't get the Overview page that I am expecting.  I get the Package page with no means of displaying the Overview.  The \<doctitle> that you have in your pom.xml should be putting the title and version on the Overview page, but for some reason, that Overview is not being generated...

For example, if you look at the JSONB apidocs, the index.html starts off with the Overview. I couldn't get that to get generated for Concurrency.
https://deploy-preview-185--jakartaee-specifications.netlify.app/specifications/jsonb/2.0/apidocs/ 

If we can't figure it out by the weekend, we'll just go with what you have and it can be corrected later.  Thanks.

Reply:
Not sure I will get a chance to investigate this before the weekend as I am putting together the Connectors PR. I will create an issue on the spec project to cover this after the milestone release.

Reply:
Sounds like a plan, @smillidge.  Thanks!